frontend/src/domains/plays/Play/Play.js

```diff
@@ -3,6 +3,7 @@ import { Link } from 'react-router-dom'
 import styled from 'styled-components'
 
 import { apiBaseURL } from '../../../services/playsRestClient'
+import { getLocaleDate } from '../../../utilities/getLocaleDate'
 import Star from '../../../components/icons/StarIcon'
 
 export default function Play({ play }) {
```

Comment:
Ist mit 'Play' immer das Spiel an sich gemacht? Mir kam spontan der Gedanke ob vielleicht 'Game' dann ein etwas selbsterklärenderer Name ist? Bei 'play' denke ich immer zuerst an das Verb 'to play' und würde eher denken, dass diese Funktion, quasi das Spiel spielt und nicht das Spiel selber ist. Ich hoffe du verstehst was ich meine :)

Reply:
Wie bereits erwähnt ist Play eine Spielepartie. Game wäre der falsche Ausdruck, da damit ein Spiel gemeint ist und nicht eine Brettspielrunde. Da es unter "Experten" ein geläufiger Term ist, würde ich es erst einmal so belassen. Verwirrend kann das definitiv sein.